delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Similar to other files, this one is vulnerable to SQL injection due to the use of raw queries without prepared statements. Utilize prepared statements and consider refactoring for object-oriented practices.